core/src/banking_stage.rs

```diff
@@ -804,7 +804,9 @@ mod tests {
     #[test]
     fn test_banking_stage_entries_only() {
         solana_logger::setup();
-        let (genesis_block, mint_keypair) = create_genesis_block(10);
+        let (mut genesis_block, mint_keypair) = create_genesis_block(10);
```

Comment:
what's this change for?

Reply:
The allocator takes longer, so the test needed more time to avoid the first slot running out of ticks.

Reply:
We may be able to rearrange some stuff so that the tick start happens closer to the processing of transactions.

Reply:
can we pre-allocate some amount in recycler?

Reply:
yea, I can try that.

Reply:
Turns out I don't need it since I added a flag to only pin when necessary.

Reply:
did that work out?

Reply:
The flag works so I didn't need to modify the test. I didn't try pre-allocating in the recycler but I didn't see an easy way to do that either.